app/statefull_flow/objective.yaml

```diff
@@ -0,0 +1,115 @@
+components:
+  schemas:
+    StatefulFlow.Objective:
+      description: >-
+        objective config such simulated user, thoughput, conncurrent connections.
+      type: object
+      properties:
+        choice:
```

Comment:
we mentioned that iperf can be mapped to this model overall.  will any of these 3 objectives map to iperf?  If yes, how?  examples?

iperf has a mode that is able to just seek max bandwidth/tput, but how would. someone map the throughput objectives defined here to iperf to make sense?  what can we measure?

Reply:
@winstonliu1111 @ajbalogh 
we could map tcp and other iperf, nperf properties in proposed model. To measure max throughput as iperf/nperf in L47 model, following are the proposal .

```
    Objective.Throughput:
      description: >-
        Throughput traffic 
      type: object
      required: [choice]
      properties:
        choice:
          description: |-
            The throughput test value or max throughput .
          type: string
          default: value
          x-field-uid: 1
          x-enum:
            value:
              x-field-uid: 1
            max_throughput:
              x-field-uid: 2
        value:
          type: integer
          x-field-uid: 2
        max_throughput:
          type: boolean
          default: 0
          x-field-uid: 3
```